site/content/en/docs/faq/_index.md

```diff
@@ -17,3 +17,9 @@ The easiest approach is to use the `docker` driver, as the backend service alway
 `none` users may want to try `CHANGE_MINIKUBE_NONE_USER=true`,  where kubectl and such will still work: [see environment variables]({{< ref "/docs/handbook/config.md#environment-variables" >}})
 
 Alternatively, configure `sudo` to never prompt for the commands issued by minikube.
+
+## How to ignore system verification?
+
+minikube's bootstrapper, [Kubeadm]({{< ref "https://github.com/kubernetes/kubeadm" >}}) verifies a list of features on the host system before installing Kubernetes. in case you get this error, and you still want to try minikube anyways despite your system's limitation you can skip the verification by starting minikube with this extra option:
```

Comment:
you can't use ({{< ref for External links

Reply:
updated

Reply:
@medyagh PTAL